MapboxVision/VisionManager/VisionManager.swift

```diff
@@ -325,7 +325,7 @@ public final class VisionManager {
     
     private func registerDefaults() {
         let defaults = UserDefaults.standard
-        defaults.setDefaultValue(true, forKey: VisionSettings.enableSync)
+        defaults.setDefaultValue(false, forKey: VisionSettings.enableSync)
```

Comment:
This line sets value only if there is no value for this key. Seems like in this case we need to set this value every time.